pkg/logs/input/docker/launcher.go

```diff
@@ -19,29 +22,38 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/logs/service"
 )
 
+const (
+	backoffInitialDuration = 1 * time.Second
+	backoffMaxDuration     = 60 * time.Second
+)
+
 // A Launcher starts and stops new tailers for every new containers discovered by autodiscovery.
 type Launcher struct {
-	pipelineProvider  pipeline.Provider
-	sources           *config.LogSources
-	services          *service.Services
-	activeSources     []*config.LogSource
-	pendingContainers map[string]*Container
-	tailers           map[string]*Tailer
-	cli               *client.Client
-	registry          auditor.Registry
-	stop              chan struct{}
+	pipelineProvider   pipeline.Provider
+	sources            *config.LogSources
+	services           *service.Services
+	activeSources      []*config.LogSource
+	pendingContainers  map[string]*Container
+	tailers            map[string]*Tailer
+	cli                *client.Client
+	registry           auditor.Registry
+	stop               chan struct{}
+	erroredContainerID chan string
```

Comment:
This name looks odd to me because the channel transport an error, would `containerToRestart` make more sense ?

Reply:
No the channel transport the `ContainerID` to restart

Reply:
Agreed, so why do we name it like so ? Could we find a better name please ?